innovation_sweet_spots/getters/inputs.py

```diff
@@ -67,10 +73,73 @@ def get_gtr_projects(fpath=GTR_PATH, fields=["id"], use_cached=True):
     return projects
 
 
+def get_cb_data(fpath=CB_PATH, cb_data_spec_path=CB_DATA_SPEC_PATH, use_cached=True):
+    """
+    Downloads Crunchbase data from Nesta database and stores it locally.
+    Function can be used from command line as follows:
+        python -c "from innovation_sweet_spots.getters.inputs import get_cb_data; get_cb_data(use_cached=False);"
+
+    Parameters
+    ----------
+    fpath : str
+        Location on disk for saving the projects table
+    cb_data_spec_path : str
+       Path to the config file that specifies which tables and columns to load in
+    use_cached: bool
+        If use_cached=True, the function will load in the local version of the dataset;
+        set use_cached=False to download (and overwrite the existing) data
+
+    Returns
+    -------
+    list of dict:
+        Dictionaries with the requested tables and paths where they have been saved.
+        The dictionaries follow the structure:
+            {"name": name of the table (corresponds to the table names from cb_data_spec_path),
+             "path": path where the table is stored,
+             "data": pandas dataframe with the data}
+    """
+    # Import specification of which tables and columns to download
+    with open(cb_data_spec_path, "r", encoding="utf-8") as yaml_file:
+        cb_tables = safe_load(yaml_file)
+        cb_table_names = list(cb_tables.keys())
+
+    tables = []
+    if not use_cached:
+        logging.info(f"Collection of business organisation data in progress")
+        con = get_engine(db_config_path)
+        # Download the specified tables one by one
+        for table in cb_table_names:
+            chunks = read_sql_table(
+                table, con, columns=cb_tables[table], chunksize=1000
+            )
+            # Combine all chunks
+            df = concat([c for c in chunks], axis=0).reset_index()
+            savepath = f"{fpath}/{table}.csv"
+            df.to_csv(savepath, index=False)
+            tables.append({"name": table, "path": savepath, "data": df})
+            logging.info(f"Collected {table} ({len(df)} rows) and stored in {savepath}")
+    else:
+        for table in cb_table_names:
+            loadpath = f"{fpath}/{table}.csv"
+            try:
+                df = read_csv(loadpath)
+                logging.info(f"Loaded in the file {loadpath}")
+            except FileNotFoundError:
+                df = []
+                logging.error(
+                    f"File {loadpath} does not exist! Set use_cached=False to download the data."
+                )
+            tables.append({"name": table, "path": loadpath, "data": df})
```

Comment:
You shouldn't allow the user to carry on through an intolerable error. Similarly to my comment in the previous PR - you should set a mode so that `use_cached=False` is switched if the path doesn't exist

Reply:
Alright! There are several files in this case, so will have to change the logic a bit (I guess, doing the existence check for each file, and if it doesn't exist, fetch it from the database)